pkg/common/tracer/traceinfo/httpstats.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 var (
-	_           HTTPStats = &httpStats{}
+	_           HTTPStats = (*httpStats)(nil)
```

Comment:
It is recommended not to put the following variables together

`var _ HTTPStats = (*httpStats)(nil)`

Reply:
> It is recommended not to put the following variables together
> 
> `var _ HTTPStats = (*httpStats)(nil)`

done